src/main/java/net/minestom/server/instance/SharedInstance.java

```diff
@@ -104,6 +104,11 @@ public boolean isInVoid(@NotNull Point point) {
         return instanceContainer.isInVoid(point);
     }
 
+    @Override
+    protected void setRegistered(boolean registered) {
+        instanceContainer.setRegistered(registered);
+    }
+
```

Comment:
@TheMode is that correct? The SharedInstance could be unregistered but the InstanceContainer can still live right?
To me, it should keep its chunk loader alive.

Reply:
I agree, it should be kept alive. It's a misunderstanding from me